_hackbars/Aryan_Bindal.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6303001)